main.go

```diff
@@ -55,6 +55,7 @@ type BuildConfig struct {
 	printSizes    string
 	cFlags        []string
 	ldFlags       []string
+	linkerScript  string
```

Comment:
This looks very similar to the following commit: https://github.com/tinygo-org/tinygo/pull/98/commits/b9c248a4911689d6dec307716e56cb30cc4a9ba1

Have you by any chance included that commit in this PR, changed it, and squashed it somewhere?
I think this is a useful change, but again, it's unrelated to rpi3 support so should stay out of this PR.

I have a suspicion that you can fix the timeouts in CI by removing these `linkerScript` changes.